heracal/__init__.py

```diff
@@ -1 +1,4 @@
-import omni, firstcal, xrfi, metrics
+import omni
+import firstcal
+import xrfi
+import metrics
```

Comment:
We need to move `xrfi` and `metrics` out of heracal, but that's not the purpose of this commit